### PR TITLE
Update default minimum viewer protocol to TLSv1.2_2021

### PR DESCRIPTION
**Pull request by venkatamutyala**
## what

By default deprecated protocols are being supported:

image

This change would disable support for viewers using TLS 1.1 and  TLS 1.0.

## why

1.0 and 1.1 are known to be deprecated/insecure. To save folks trouble by their security teams using the latest version seems the most appropriate. 

## references

https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/secure-connections-supported-viewer-protocols-ciphers.html

### CODE REVIEW

**Comment by joe-niland** [comment]
Thanks @venkatamutyala 

Could you please run the following and commit the result?

```
make init
make readme
```

**Comment by venkatamutyala** [comment]
Done. Let me know if you folks need anything else.

File: variables.tf [hunk 1]
```diff
@@ -215,7 +215,7 @@ variable "price_class" {
 variable "viewer_minimum_protocol_version" {
   type        = string
   description = "The minimum version of the SSL protocol that you want CloudFront to use for HTTPS connections."
```
Comment (joe-niland) [inline variables.tf:217]:
```suggestion
  description = "The minimum version of the SSL protocol that you want CloudFront to use for HTTPS connections. This is ignored if the default CloudFront certificate is used."
```

**Review by joe-niland — COMMENTED** [review]
One suggestion

**Comment by joe-niland** [comment]
Thanks @venkatamutyala the latest change will require the readme to be updated again

**Review by nitrocode — APPROVED** [review]

**Comment by joe-niland** [comment]
Thanks for your contribution @venkatamutyala

**Pull request merged**